.github/workflows/examples-old-nodes.yml

```diff
@@ -73,3 +73,74 @@ jobs:
         run: pnpm turbo --filter ./examples/${{ matrix.dir }} test-dev
       - name: Run test-start
         run: pnpm turbo --filter ./examples/${{ matrix.dir }} test-start
+
+  e2e-old-nodes-node-14:
```

Comment:
is this worth it to retain the node-14 workflow?

pnpm@8 is >=node16 so i split this workflow into 2

Reply:
node 14 goes end of life next month so I think it's ok...

Reply:
aight i'll just nuke node14 then :)